atlassian/bitbucket/__init__.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, url, *args, **kwargs):
                 "2.0" if "cloud" in kwargs and kwargs["cloud"] else "1.0"
             )
         if "cloud" in kwargs:
-            kwargs["api_root"] = "" if "api.bitbucket.org" in url else "api"
+            kwargs["api_root"] = "" if "api.bitbucket.org" in url else "rest/api"
```

Comment:
This is incorrect, the `api_root` for Bitbucket **Cloud** is either `api.bitbucket.org` or `bitbucket.org/api`, there's no `/rest` in there

Reply:
Of course there should no `rest` be added.